Assets/Project/Modules/VFX/Anchor/Throw/Scripts/AnchorThrowHeadFollower.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using UnityEngine;
+
+namespace Project.Modules.VFX.Anchor.Throw.Scripts
```

Comment:
The namespaces must start with "Popeye". not "Project".
And you should also remove the "Scripts" part

Reply:
Done 👌